<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshMapping2D</class>
 <widget class="QDialog" name="MeshMapping2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>2D Mesh Mapping</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QPushButton" name="rasterSelectButton">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="noDataValueLabel">
     <property name="text">
      <string>     Set NoData values to:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QRadioButton" name="staticValueButton">
     <property name="text">
      <string>Map to static value:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="rasterPathEdit"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QRadioButton" name="rasterValueButton">
     <property name="text">
      <string>Map mesh based on raster file</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="noDataValueEdit">
     <property name="text">
      <string>0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="staticValueEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="rasterPathLabel">
     <property name="text">
      <string>      Raster file:</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="newNameEdit">
     <property name="text">
      <string>MappedMesh</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="newNameLabel">
     <property name="text">
      <string>Name of new mesh:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rasterValueButton</tabstop>
  <tabstop>rasterPathEdit</tabstop>
  <tabstop>rasterSelectButton</tabstop>
  <tabstop>noDataValueEdit</tabstop>
  <tabstop>staticValueButton</tabstop>
  <tabstop>staticValueEdit</tabstop>
  <tabstop>newNameEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MeshMapping2D</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MeshMapping2D</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
